.github/workflows/cspellcdict.txt

```diff
@@ -3162,3 +3182,22 @@ Gmail
 vmf
 mdl
 html
+VANTS
+Dont
+PHQ
+PCL
+vis
+ICD
+ZZ
+LSV
+xlsb
```

Comment:
This isn't meant to be xlsx is it?

Reply:
Yes.  xlsx.  I will correct before merge.